src/lib/import-export/tests/export/export-manager.test.ts

```diff
@@ -1,96 +1,63 @@
-import fs from 'fs/promises';
-import path from 'path';
 import { exportBackup } from '../../export/export-manager';
-import { ExportOptions, ExporterOption, BackupContents } from '../../export/types';
-
-jest.mock( 'fs/promises' );
-jest.mock( 'path' );
-jest.mock( '../../export/validators/wordpress-validator' );
-jest.mock( '../../export/exporters' );
+import { ExportOptions, NewExporter } from '../../export/types';
 
 describe( 'exportBackup', () => {
-	const mockSitePath = '/mock/site/path';
-	const mockExportOptions: ExportOptions = {
-		sitePath: mockSitePath,
-		backupFile: '/mock/backup/path/backup.tar.gz',
-		includes: {
-			database: true,
-			uploads: true,
-			plugins: true,
-			themes: true,
-		},
-	};
-
-	const mockFiles = [
-		'wp-config.php',
-		'wp-content/uploads/image.jpg',
-		'wp-content/plugins/plugin.php',
-		'wp-content/themes/theme/style.css',
-	];
-
-	const expectedFiles = mockFiles.map( ( file ) => `${ mockSitePath }/${ file }` );
-
+	let mockExportOptions: ExportOptions;
 	beforeEach( () => {
-		jest.clearAllMocks();
-		( fs.readdir as jest.Mock ).mockResolvedValue(
-			mockFiles.map( ( file ) => ( {
-				isFile: () => true,
-				path: mockSitePath,
-				name: file,
-			} ) )
-		);
-		( path.join as jest.Mock ).mockImplementation( ( ...args ) => args.join( '/' ) );
+		mockExportOptions = {} as ExportOptions;
+		console.log = jest.fn(); // Mock console.log
```

Comment:
```suggestion
		console.log = jest.fn(); 
```

I am not completely sure the comment is neccessary, it seems pretty self-explanatory. What do you think?